src/target/stm32h7.c

```diff
@@ -257,9 +257,20 @@ void stm32h7_configure_wdts(target_s *const target)
 bool stm32h7_probe(target_s *target)
 {
 	const adiv5_access_port_s *const ap = cortex_ap(target);
+	DEBUG_TARGET("%s: AP partno %X, part_id %X\n", __func__, ap->partno, target->part_id);
```

Comment:
This reads like some residual debugging stuff?

If something like this is actually warranted, we'd suggest putting it into cortexm.c in the handling case for STM32 parts, and making the print `DEBUG_TARGET("%s: AP part id: %04x, part id: %04x\n", __func__, ap->partno, target->part_id);` or so.

Essentially, please use lower-case hex for code base consistency, and force the formatted part number to be 4 digits long to get a consistent output. Having the display for the STM32 parts is probably good insurance given all the problems with their part numbering inconsistencies, but it should then be in the Cortex-M handling layer, rather than duplicated across all the target support implementations.